content/id/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -0,0 +1,253 @@
+
+```shell
+kubectl create secret generic test-secret --from-literal='username=my-app' --from-literal='password=39528$vdg7Jb'
+```
+
+Tentu saja ini lebih mudah. Pendekatan yang mendetil setiap langkah di atas bertujuan untuk mendemonstrasikan apa yang sebenarnya terjadi.
+
+
+## Membuat Pod yang memiliki akses ke data Secret melalui Volume
+
+Berikut ini adalah berkas konfigurasi yang dapat kamu gunakan untuk membuat Pod:
+
+{{< codenew file="pods/inject/secret-pod.yaml" >}}
+
+1. Membuat Pod:
+
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/inject/secret-pod.yaml
+   ```
+
+1. Verifikasikan apakah Pod kamu sudah berjalan:
+
+   ```shell
+   kubectl get pod secret-test-pod
+   ```
+
+   Hasil keluaran:
+   ```
+   NAME              READY     STATUS    RESTARTS   AGE
+   secret-test-pod   1/1       Running   0          42m
+   ```
+
+1. Gunakan _shell_ untuk masuk ke dalam Container yang berjalan di dalam Pod kamu:
+   ```shell
+   kubectl exec -i -t secret-test-pod -- /bin/bash
+   ```
+
+1. Data Secret terekspos ke Container melalui Volume yang dipasang (_mount_) pada
+`/etc/secret-volume`.
+
+   Di dalam _shell_ kamu, tampilkan berkas yang ada di dalam direktori `/etc/secret-volume`:
+   ```shell
+   # Jalankan ini di dalam shell dalam Container
+   ls /etc/secret-volume
+   ```
+   Hasil keluaran menampilkan dua buah berkas, masing-masing untuk setiap data Secret:
+   ```
+   password username
+   ```
+
+1. Di dalam _shell_ kamu, tampilkan konten dari berkas `username` dan `password`:
+   ```shell
+   # Jalankan ini di dalam shell dalam Container
+   echo "$( cat /etc/secret-volume/username )"
+   echo "$( cat /etc/secret-volume/password )"
+   ```
+   Hasil keluarannya adalah nama pengguna dan kata sandi kamu:
+   ```
+   my-app
+   39528$vdg7Jb
+   ```
+
+## Mendefinisikan variabel lingkungan Container menggunakan data Secret
+
+### Mendefinisikan variabel lingkungan Container menggunakan data dari Secret tunggal
+
+*  Definisikan variabel lingkungan sebagai pasangan _key-value_ pada Secret:
+
+   ```shell
+   kubectl create secret generic backend-user --from-literal=backend-username='backend-admin'
+   ```
+
+*  Tentukan nilai `backend-username` yang didefinisikan di Secret ke variabel lingkungan `SECRET_USERNAME` di dalam spesifikasi Pod.
+
+   {{< codenew file="pods/inject/pod-single-secret-env-variable.yaml" >}}
+
+*  Membuat Pod:
+
+   ```shell
+   kubectl create -f https://k8s.io/examples/pods/inject/pod-single-secret-env-variable.yaml
+   ```
+
+*  Di dalam _shell_ kamu, tampilkan konten dari variabel lingkungan `SECRET_USERNAME` dari Container
+
+   ```shell
+   kubectl exec -i -t env-single-secret -- /bin/sh -c 'echo $SECRET_USERNAME'
+   ```
+
+   Hasil keluarannya
+   ```
+   backend-admin
+   ```
+
+### Mendefinisikan variabel lingkungan Container dengan data dari banyak Secret
```

Comment:
we can use "multipel". It is Indonesian word.
```suggestion
### Mendefinisikan variabel lingkungan Container dengan data dari Secret multipel
```

Reply:
Secret multipel or multipel Secret? I prefer the later

Reply:
Both are ok, just try to make it consistent.